pkg/dom0-ztools/rootfs/etc/init.d/010-eve-cgroup

```diff
@@ -5,9 +5,13 @@ if test -f /proc/vmcore; then
     exit 0;
 fi
 
-default_cgroup_memory_limit=838860800000 #800M
+virttype=`cat /run/eve-virt-type`
+default_cgroup_memory_limit=838860800 #800M
```

Comment:
You can add it to an else in the if below....

Reply:
ok